corehq/apps/reports/dbaccessors.py

```diff
@@ -11,5 +12,19 @@ def get_exports(domain):
         "couchexport/saved_export_schemas",
         startkey=startkey,
         endkey=endkey,
-        include_docs=True,
+        include_docs=include_docs,
```

Comment:
@dannyroberts not sure how strictly you were thinking of adhering to the convention of not parameterizing the `include_docs` variable, but i thought i'd flag it. i imagine it will be hard to enforce over time